setup.py

```diff
@@ -128,6 +128,7 @@ def cuaev_kwargs():
         'torch',
         'lark-parser',
         'requests',
+        'h5py',
```

Comment:
Why h5py is a dependency of torchani, and why it is related to this PR?

Reply:
removed